lib/components/viewers/stop-time-cell.tsx

```diff
@@ -1,16 +1,20 @@
-import 'moment-timezone'
-import { FormattedMessage, FormattedTime } from 'react-intl'
+import { format, getTimezoneOffset, utcToZonedTime } from 'date-fns-tz'
+import { FormattedMessage } from 'react-intl'
+import addDays from 'date-fns/addDays'
 // TODO: typescript core-utils, add common types (pattern, stop, configs)
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
```

Comment:
Same as above -- I believe we can remove this line now that core-utils has been upgraded.